App crashes regularly on A14

> [Device Brand]: google
[Device Model]: Pixel 7
[Android Version]: 14
[Android API Level]: 34
[Process ID]: 3149
[Package Name]: com.bnyro.clock
[Version Name]: 6.4
[Version Code]: 13
[Target SDK]: 33
[Min SDK]: 21
[Error Type]: JVM
[Crash Time]: 2023-12-06T10:09:19.473
[Stack Trace]:
java.lang.RuntimeException: Unable to create application com.bnyro.clock.App: java.lang.IllegalStateException: SharedPreferences in credential encrypted storage are not available until after user (id 0) is unlocked
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7003)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2236)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8177)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: java.lang.IllegalStateException: SharedPreferences in credential encrypted storage are not available until after user (id 0) is unlocked
	at android.app.ContextImpl.getSharedPreferences(ContextImpl.java:604)
	at android.app.ContextImpl.getSharedPreferences(ContextImpl.java:587)
	at android.content.ContextWrapper.getSharedPreferences(ContextWrapper.java:221)
	at com.bnyro.clock.App.onCreate(SourceFile:851)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1316)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6998)

> ... 9 more

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -57,9 +57,6 @@
             <intent-filter>
                 <action android:name="android.intent.action.BOOT_COMPLETED" />
             </intent-filter>
-            <intent-filter>
-                <action android:name="android.intent.action.LOCKED_BOOT_COMPLETED" />
-            </intent-filter>
         </receiver>
 
         <service

